electron/main/index.ts

```diff
@@ -35,13 +39,21 @@ if (!app.requestSingleInstanceLock()) {
 // Read more on https://www.electronjs.org/docs/latest/tutorial/security
 // process.env['ELECTRON_DISABLE_SECURITY_WARNINGS'] = 'true'
 
+contextMenu({
+  showSaveImageAs: true,
+});
+
 let win: BrowserWindow | null = null;
 // Here, you can also use other preload
 const preload = join(__dirname, "../preload/index.js");
 const url = process.env.VITE_DEV_SERVER_URL;
 const indexHtml = join(process.env.DIST, "index.html");
 
 async function createWindow() {
+  installExtension(REACT_DEVELOPER_TOOLS)
```

Comment:
Can u make it install only for dev but not prod?

Reply:
turns out installing react dev tools in electron is just broken as a whole right now, i got rid of the package

Reply:
lol classic react ecosystems